Userland/Libraries/LibGfx/Bitmap.cpp

```diff
@@ -140,6 +141,18 @@ ErrorOr<NonnullRefPtr<Bitmap>> Bitmap::try_load_from_file(StringView path, int s
     return try_load_from_fd_and_close(fd, path);
 }
 
+ErrorOr<NonnullRefPtr<Bitmap>> Bitmap::try_load_from_stream(NonnullOwnPtr<Core::Stream::File> file, StringView path)
```

Comment:
This name is a bit misleading since we're not actually doing any kind of streamed load. We're actually mapping a *file* that happens to be inside something called `Core::Stream::File`.

Reply:
I added the function as an overload of `try_load_from_file`, hope that it's correct